pages/common/ionic.md

```diff
@@ -0,0 +1,31 @@
+# ionic
+
+> Complete open source SDK to build top quality native and progressive web apps using Angular JS and Apache Cordova.  
```

Comment:
We tend to omit adjectives ("complete", "open source", "top quality") in the main description. Stick to describing what it is and what it does :)

Btw, "progressive web app" (PWA) seems to be kind of a specialized term, which is [explained even to ionic users](http://blog.ionic.io/what-is-a-progressive-web-app/). I think it's not generic/informative enough to be included here. Maybe "top quality native and progressive web apps" --> "native-like mobile web apps"?